packages/ra-auth-msal/src/authProvider.ts

```diff
@@ -38,6 +38,8 @@ export type MsalAuthProviderParams = {
   redirectOnCheckAuth?: boolean;
 };
 
+const MSAL_REDIRECT_KEY = "_ra_msal_key";
```

Comment:
```suggestion
const MSAL_REDIRECT_KEY = "_ra_msal_redirect_key";
```

(Better in case we need to add a second key later)